src/signature/generalized_xmss.rs

```diff
@@ -666,17 +669,15 @@ mod tests {
         assert_eq!(rho1, rho2);
     }
 
-    /*#[test]
-    pub fn test_large_base_sha() {
+    #[test]
+    pub fn test_large_base_poseidon() {
         // Note: do not use these parameters, they are just for testing
-        type PRF = ShaPRF<24, 8>;
-        type TH = ShaTweak192192;
-
-        // use chunk size 8
-        type MH = ShaMessageHash<24, 8, 32, 8>;
-        const TARGET_SUM: usize = 1 << 12;
+        type PRF = ShakePRFtoF<4, 4>;
```

Comment:
Why can't we use this? We want to be as close as possible to the old test configuration I think

```suggestion
        type PRF = ShakePRFtoF<24, 8>;
```